notebooks/generate_rendered_notebook_for_docs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+THIS_DIR="$( cd "$(dirname "$0")"; pwd -P )"
+
+jupyter nbconvert --to notebook --execute --output="$THIS_DIR"/../docs/iceflow-example.ipynb "$THIS_DIR"/iceflow-example.ipynb
```

Comment:
```suggestion
jupyter nbconvert --to notebook --execute --output="${THIS_DIR}/../docs/iceflow-example.ipynb" "${THIS_DIR}/iceflow-example.ipynb"
```

Quoting just the variable part of the arg is confusing to me :)